src/config/index.ts

```diff
@@ -235,7 +235,7 @@ export function valueIsValid<T extends SchemaTypeKey>(
 
 async function getSchema<T extends Schema.SchemaKey>(
 	key: T,
-	$db: Transaction,
+	$db: typeof db.models,
```

Comment:
This change is necessary to pass `strictBindCallApply` - it used to work because `db.models.bind(db)` would convert to `any` and so would "pass" type checking